lib/spree_google_base/engine.rb

```diff
@@ -19,7 +19,8 @@ class Engine < Rails::Engine
         ['g:condition', 'google_base_condition'],
         ['g:availability', 'google_base_availability'],
         ['g:google_product_category', 'google_base_product_category'],
-        ['g:item_group_id', 'google_base_item_group_id']
+        ['g:item_group_id', 'google_base_item_group_id'],
+        ['g:item_color', 'google_base_item_color']
```

Comment:
i was looking at the documentation (https://support.google.com/merchants/answer/6324487) and it seems like the attribute should be called `g:color`